gallery/src/components/NxModal/NxModalFormErrorExample.tsx

```diff
@@ -6,9 +6,11 @@
  */
 import React, {useState} from 'react';
 
-import {NxModal, NxFontAwesomeIcon, NxTextInput, NxLoadError, NxButton} from '@sonatype/react-shared-components';
+import {NxModal, NxFontAwesomeIcon, NxTextInput, NxLoadError, NxButton, nxTextInputStateHelpers}
+  from '@sonatype/react-shared-components';
 import {faAngry} from '@fortawesome/free-solid-svg-icons';
-import { initialState, userInput } from '@sonatype/react-shared-components/components/NxTextInput/stateHelpers';
+
+const { initialState, userInput } = nxTextInputStateHelpers;
```

Comment:
```suggestion
const {initialState, userInput} = nxTextInputStateHelpers;
```

Reply:
> const {initialState, userInput} = nxTextInputStateHelpers;

That's not how I do it.

Reply:
Then line 9 and line 11 need be changed. For consistency.

Reply:
> Then line 9 and line 11 need be changed. For consistency.

Line 11 (and 7) are arguably out of scope but here you go: fd305b7b

Reply:
> > Then line 9 and line 11 need be changed. For consistency.
> 
> Line 11 (and 7) are arguably out of scope but here you go: [fd305b7](https://github.com/sonatype/sonatype-react-shared-components/commit/fd305b7b23d30bc3055d8361b8cf981f1cd2cbda)

Thanks for humoring my obsesssion!